kafka/kafka.go

```diff
@@ -103,6 +104,20 @@ func GetInserterConfig() (*ReaderConfig, *WriterConfig) {
 	return readerConfig, writerConfig
 }
 
+//GetInserterConfigNew returns the Reader topics from kafka for Inserters
+func GetInserterConfigNew() *ReaderConfig {
```

Comment:
This should just be `GetInserterConfig`
The old implementation should be renamed to `DEPRECATED_GetInserterConfig` or something along that line